### PR TITLE
Change the color and visibility of the icons on the status bar for the Overview and MyPICOS 

### PR DESCRIPTION
**Pull request by FaysalSolutions**
1. Match the color of the statusbar to the content
2. Change the Icons of the statusbar to be more visible
3. Disable the shadows of the app bar

So:
- fixes issue #136 

4. In this pull request, I have set the size of the app bar on the LoginScreen to zero:

image

### CODE REVIEW

File: lib/screens/login_screen.dart [hunk 1]
```diff
@@ -86,103 +86,119 @@ class _LoginScreenState extends State<LoginScreen>
...
+                  text: TextSpan(
+                    style: const TextStyle(
+                      height: 2,
+                      color: Colors.black,
                     ),
-                  ],
+                    children: <TextSpan>[
+                      TextSpan(
+                        text:
+                            '${AppLocalizations.of(context)!.welcomeToPICOS},\n',
```
Comment (S1rDan) [inline lib/screens/login_screen.dart:123]: Try not to make lines above 80 characters.

**Review by S1rDan — CHANGES_REQUESTED** [review]

File: lib/screens/home_screen/overview/overview.dart [hunk 2]
```diff
@@ -16,43 +16,87 @@
...
+  }
+
+  void _handleScroll() {
+    final double appBarHeight = AppBar().preferredSize.height;
+    final double statusBarHeight = MediaQuery.of(context).padding.top;
+
+    setState(() {
+      _isScrolled =
+          _scrollController.offset >= (appBarHeight + statusBarHeight);
+    });
```
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/home_screen/overview/overview.dart]: Resolved in 4f3ab78

File: lib/screens/home_screen/overview/overview.dart [hunk 3]
```diff
@@ -16,43 +16,87 @@
...
+
+  void _setSystemNavigationBarColor(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(
+      SystemUiOverlayStyle(
+        statusBarColor: _isScrolled
+            ? Theme.of(context).extension<GlobalTheme>()!.darkGreen1
+            : Theme.of(context).bottomNavigationBarTheme.backgroundColor ??
+                Theme.of(context).canvasColor,
+        statusBarIconBrightness:
+            _isScrolled ? Brightness.light : Brightness.dark,
```
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/home_screen/overview/overview.dart]: Resolved in 61e017e8c3fefb67dc5eecd88e370f69e9372a8e

File: lib/screens/home_screen/home_screen.dart [hunk 4]
```diff
@@ -89,10 +89,12 @@ class _HomeScreenState extends State<HomeScreen> {
       );
     }
 
-    return PicosScreenFrame(
-      body: const PicosMenu(),
-      title: 'MyPICOS',
-      bottomNavigationBar: bottomNavigationBar,
+    return SafeArea(
+      child: PicosScreenFrame(
```
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/home_screen/home_screen.dart]: Resolved in 13a3e1dd5922e428e24519eeccf3cc14f0a4a23b

File: lib/screens/home_screen/overview/overview.dart [hunk 5]
```diff
@@ -16,43 +16,87 @@
...
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController.addListener(_handleScroll);
+  }
+
+  void _handleScroll() {
+    final double appBarHeight = AppBar().preferredSize.height;
+    final double statusBarHeight = MediaQuery.of(context).padding.top;
```
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/home_screen/overview/overview.dart]: Resolved in 815ddb2de19465e6d3b91276e996e67b7bbca35e

File: lib/screens/login_screen.dart [hunk 6]
```diff
@@ -86,103 +86,119 @@ class _LoginScreenState extends State<LoginScreen>
...
-                      text:
-                          '${AppLocalizations.of(context)!.welcomeToPICOS},\n',
-                      style: const TextStyle(
-                        fontWeight: FontWeight.bold,
-                      ),
-                    ),
-                    TextSpan(
-                      text: AppLocalizations.of(context)!
-                          .thankYouForParticipation,
+    _setSystemNavigationBarColor(context);
```
[the remark this answers is not in the document]
Comment (FaysalSolutions) [reply lib/screens/login_screen.dart]: Resolved in 25f3d72bca2f8adbb29a8a1adb91c4084ececba5

File: lib/screens/login_screen.dart (hunk 1 above)
Comment (FaysalSolutions) [reply lib/screens/login_screen.dart]: Resolved in e9c442a7f063ca9fcd15e451ed0870d25ed632c2

**Pull request closed without merging**